internal/mutators/podtemplatespec.go

```diff
@@ -0,0 +1,113 @@
+package mutators
+
+import (
+	"fmt"
+
+	"github.com/kanopy-platform/hedgetrimmer/internal/limitrange"
+	"github.com/kanopy-platform/hedgetrimmer/internal/quantity"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type PodTemplateSpec struct{}
+
+func NewPtsMutator() PodTemplateSpec {
+	return PodTemplateSpec{}
+}
+
+func (p PodTemplateSpec) Mutate(inputPts corev1.PodTemplateSpec, lri corev1.LimitRangeItem) (corev1.PodTemplateSpec, error) {
```

Comment:
I ended up adding it to `pkg/admission/handlers/interface.go`. I think that's where it will be consumed by the handlers as input arguments so I think that's the spot instead of `internal/admission` as previously discussed.